### PR TITLE
hunt updates and schedtasks rewrite

### PR DESCRIPTION
**Pull request by 5ynax**

### CODE REVIEW

File: Plugins/Hunt/Hunt-MaliciousScheduledTaskInfo.ps1 [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+    Plugin-Name: Hunt-MaliciousScheduledTaskInfo.ps1
+    
+.Description
+
+    Hunts for malicious scheduled tasks and performs analysis to discover anomalies
+
+.EXAMPLE
+
+    Set ComputerName <list of computers>
+    Set HuntName UserAccountsHunt
```
Comment (valrkey) [inline Plugins/Hunt/Hunt-MaliciousScheduledTaskInfo.ps1]: `ScheduledTaskHunt`

File: Plugins/Persistence/Collect-ScheduledTaskDetails.ps1 [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+    does not collect the scheduled task files themselves
+
+    To collect the scheduled task configuration files use the Collect-ScheduledTasks
+    plugin.
+
+.EXAMPLE
+
+    Power-Response Execution
+
+    Set ComputerName Test-PC
```
Comment (valrkey) [inline Plugins/Persistence/Collect-ScheduledTaskDetails.ps1]: `Test-PC` referenced directly in favor of `<list of computers>`? Also no HuntName?

Comment (5ynax) [reply Plugins/Persistence/Collect-ScheduledTaskDetails.ps1]: collect plugin, not hunt.

File: Plugins/Hunt/Hunt-MaliciousScheduledTaskInfo.ps1 (hunk 1 above)
Comment (5ynax) [reply Plugins/Hunt/Hunt-MaliciousScheduledTaskInfo.ps1]: fixed

**Pull request merged**